doc/active_docs/Service Management API (on-premises).json

```diff
@@ -56,7 +56,7 @@
               "name": "user_id",
               "dataType": "string",
               "paramType": "query",
-              "description": "User id. String identifying an end user. Required only when the application is rate limiting end users. The End User plans feature is not available in all 3scale plans."
+              "description": "User id. String identifying an end user. Required only when the application is rate limiting end users."
```

Comment:
Shouldn't we remove this?

Reply:
IDK.. I think it's just a mention of EndUsers but it's not directly related

Reply:
I agree with Hery. This user_id is to identify and EndUser, which we are removing.

Reply:
Also removing the parameter for it in the controller